### PR TITLE
Add two photon imaging

### PR DESCRIPTION
**Pull request by weiglszonja**
Add `ScanImage` data to two photon conversion:
- uses `ScanImageSinglePlaneMultiFileImagingInterface` to convert the multi-file (buffered) format with single and dual channel setup
- add basic metadata and placeholder for metadata we need from Shiva

### CODE REVIEW

**Comment by weiglszonja** [comment]
Stub is [here]([2620749R2_231211.nwb](https://us-east-2.console.aws.amazon.com/s3/object/catalystneuro-processing?region=us-east-2&bucketType=general&prefix=dombeck-lab-to-nwb/2620749R2_231211.nwb))

This is only ScanImage, the suite2p output will be added in a follow-up.

File: src/dombeck_lab_to_nwb/nagappan_embargo_2023/metadata/nagappan_embargo_2023_ophys_metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Ophys:
+  TwoPhotonSeries:
+    - name: TwoPhotonSeriesChannel1
+      description: Imaging data from Green channel recorded from cell bodies located in the striatum acquired via ScanImage.
+    - name: TwoPhotonSeriesChannel2
+      description: Imaging data from Red channel recorded from axons located in the striatum acquired via ScanImage.
+  ImagingPlane:
+    - name: ImagingPlaneChannel1
+      excitation_lambda: 920.0  # in nm
+      location: cell bodies located in the striatum
```
[the remark this answers is not in the document]
Comment (weiglszonja) [reply src/dombeck_lab_to_nwb/nagappan_embargo_2023/metadata/nagappan_embargo_2023_ophys_metadata.yaml:10]: hmm yeah, as I remember it should follow the Allen Brain Atlas naming

File: src/dombeck_lab_to_nwb/nagappan_embargo_2023/metadata/nagappan_embargo_2023_ophys_metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      optical_channel:
+        - name: Channel 1
+          description: The optical channel used to record cell bodies located in the striatum.
+      indicator: unknown # TODO collect from Shiva
+    - name: ImagingPlaneChannel2
+      excitation_lambda: 1070.0 # in nm
+      location: axons located in the striatum
```
Comment (weiglszonja) [inline src/dombeck_lab_to_nwb/nagappan_embargo_2023/metadata/nagappan_embargo_2023_ophys_metadata.yaml:17]: STR

Comment (weiglszonja) [inline src/dombeck_lab_to_nwb/nagappan_embargo_2023/metadata/nagappan_embargo_2023_ophys_metadata.yaml:17]:
```suggestion
      location: STR
```

**Review by alessandratrapani — APPROVED** [review]

**Pull request merged**